### ISSUE TITLE
Indentation and Autosearch for auto-instance

### ISSUE THREAD

**Issue by markusdd** [eirikpre/VSCode-SystemVerilog#114]
Hi. I really like this plugin and it is more comfortable to use than verilog-hdl in my opinion.

For the auto-instance feature I think 2 things would be helpful:

a) fix indentation:

![image](https://user-images.githubusercontent.com/25175069/107891835-67bd5f80-6f21-11eb-8c8a-e77e78cf22b3.png)

b) provide auto-complete when entering the module name. e.g. when I just typ 'syn' I would expect svb_sync to show up

**Comment by eirikpre** [comment]
Yeah, I know, the instantiate feature needs a small makeover to be a real quality feature.

Keeping this issue open as a means of tracking it.
